command_line/powder_calibrate.py

```diff
@@ -349,20 +363,23 @@ def __init__(
         self.detector = start_geometry.detector
         self.calibrant = calibrant
         self.coarse_geom = coarse_geom
+
         self.fig, self.ax = self.set_up_figure()
         self.calibrant_image = self.calibrant_rings_image()
-        self.beam_fast_slider = self.make_slider("fast")
-        self.beam_slow_slider = self.make_slider("slow")
+
+        self.beam_fast_slider = self._make_slider("fast")
+        self.beam_slow_slider = self._make_slider("slow")
+        self.distance_slider = self._make_slider("distance")
 
     def __repr__(self):
```

Comment:
```suggestion
    def __str__(self):
```

Reply:
Can you test that this works okay?

Reply:
I got sidetracked. 
I did move `__repr__`s  to `__str__`s but then I got hit by loss aversion bias and decided I also want to have `__repr__`s.